service/src/main/kotlin/io/provenance/explorer/grpc/v1/AccountGrpcClient.kt

```diff
@@ -107,4 +112,13 @@ class AccountGrpcClient(channelUri : URI) {
                 .setDelegatorAddress(delAddr)
                 .build())
 
+    fun getBankParams() = bankClient.params(BankOuterClass.QueryParamsRequest.newBuilder().build())
```

Comment:
Not a huge fan of have the param queries coming from the `AccountGrpcClient`. Mainly because these clients have queries tailored to the data thats being pulled. 

Bank and auth params are fine here, but I would move dist and staking to `ValidatorGrpcClient` to remian semi consistent. Mint is fine here too for now. We dont do much with mint.